cli/dumpTree.py

```diff
@@ -140,18 +142,51 @@ def dump(input_file, output_file):
 
     vertices_dtype = np.dtype([("eventID","u4"),("x_vert","f4"),("y_vert","f4"),("z_vert","f4")])
 
+    with h5py.File(output_file, 'w') as f:
+        f.create_dataset('trajectories', (0,), dtype=trajectories_dtype, maxshape=(None,))
+        f.create_dataset('segments', (0,), dtype=segments_dtype, maxshape=(None,))
+        f.create_dataset('vertices', (0,), dtype=vertices_dtype, maxshape=(None,))
+
     segments_list = []
     trajectories_list = []
     vertices_list = []
 
-    for jentry in range(entries):
-        print(jentry)
+    for jentry in tqdm(range(entries+1)):
```

Comment:
Why the +1 here?

Reply:
This is because I made it so it saves a batch of trajectories every now and then within the loop. For the last iteration, it will always save the batch to the file. I could make this less weird if you'd like (just make a dedicated function for writing to the file and then call it after the loop exits).